wcmc_components/lib/wcmc_components/filterable/serializer.rb

```diff
@@ -35,6 +35,8 @@ def serialize_relation_for_table(relation)
       def serialize_item_for_table(item)
         {
           page_url: item.table_page_path,
```

Comment:
In the frontend it needs pageUrl, but here it's page_url. This is why I was slightly confused. Is something else mapping between the two, or does this need updating to match the frontend?

In which case archive_url and edit_url will also both need to be camelCase.